MeshLib/MeshSurfaceExtraction.h

```diff
@@ -72,6 +74,8 @@ class MeshSurfaceExtraction
     static void get2DSurfaceElements(
         const std::vector<MeshLib::Element*>& all_elements,
         std::vector<MeshLib::Element*>& sfc_elements,
+        std::vector<std::size_t>& element_id_map,
+        std::vector<std::size_t>& face_id_map,
```

Comment:
These maps are mapping from which mesh to which mesh? Needs a documentation or a more descriptive name.